src/components/vm/filesystems/vmFilesystemsCard.jsx

```diff
@@ -122,7 +122,6 @@ export const VmFilesystemActions = ({ connectionName, memory, memoryBacking, obj
 const VmFilesystemAddModal = ({ connectionName, memory, memoryBacking, objPath, setIsOpen, vmName, vmState }) => {
```

Comment:
"memoryBacking" is not not used anywhere in the whole file

Reply:
Removed - thanks!